### DISCUSSION TITLE
Daqmx Input/Outputs not added to workflow

### DISCUSSION THREAD

**Discussion by jimenotis** [bonsai-rx/bonsai#1885] [Q&A]
Hello, I am just starting to use BONSAI. I am trying to connect a NI-USB 6215 card. The card driver is properly installed. However when I try to add an input or output to the workflow I get the following error. Am I missing something? Thanks!

![image](https://github.com/bonsai-rx/bonsai/assets/174745227/716b54d5-7bc5-4863-9889-f02c072a9c21)

**Comment by bruno-f-cruz** [comment]
Can you try installing the daqmx version 19.1?
